backend/src/routes/api/status/index.ts

```diff
@@ -34,15 +34,11 @@ const status = async (
       adminGroup,
     );
     const adminUsers = (adminGroupResponse.body as groupObjResponse).users;
-    if (!adminUsers || !adminUsers?.length) {
-      console.log('Warning: No admin user in admin groups');
-    } else {
-      isAdmin = adminUsers.includes(userName);
-    }
+    isAdmin = adminUsers.includes(userName);
   } catch (e) {
     console.log('Failed to get groups: ' + e.toString());
   }
-
+  fastify.kube.coreV1Api.getAPIResources();
```

Comment:
Don't we need an await or then?  I think getApiResources returns a promise.

Reply:
Removed this code it's not doing anything.

Reply:
Removed this code it's not doing anything.